deeprank/learn/NeuralNet.py

```diff
@@ -35,8 +35,9 @@ def __init__(self, data_set, model,
                  chain2='B',
                  cuda=False, ngpu=0,
                  plot=True,
-                 save_hitrate=True,
+                 save_hitrate=False,
                  save_classmetrics=False,
+                 benchmark=False,
```

Comment:
can we had 'benchmark' in the docstring. It's also sort of an unclear name, so we could think of adopting a more self describing name

Reply:
Sure, 

Is `benchmark_mode=False` better suited ?